Cabal/Distribution/Simple.hs

```diff
@@ -104,6 +104,7 @@ import Distribution.Compat.Directory        (makeAbsolute)
 import Distribution.Compat.Environment      (getEnvironment)
 import Distribution.Compat.GetShortPathName (getShortPathName)
 
+import qualified Data.ByteString.Lazy as B
```

Comment:
as LBS or BSL. We use both, but LBS is more common. (I haven't got to normalise that yet).